components/widget/ChatInput.tsx

```
@@ -6,10 +6,10 @@ import { Checkbox } from "@/components/ui/checkbox";
 import { Textarea } from "@/components/ui/textarea";
 import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@/components/ui/tooltip";
 import ShadowHoverButton from "@/components/widget/ShadowHoverButton";
-import { useScreenshotStore } from "@/components/widget/widgetState";
+import { useDraftStore, useScreenshotStore } from "@/components/widget/widgetState";
 import { captureExceptionAndLog } from "@/lib/shared/sentry";
 import { cn } from "@/lib/utils";
-import { closeWidget, sendScreenshot } from "@/lib/widget/messages";
+import { closeWidget, sendScreenshot, sendScreenshotWithErrorHandling } from "@/lib/widget/messages";
```

ESLint

[error] 'sendScreenshotWithErrorHandling' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)